lib/wallet/integration/src/Test/Integration/Scenario/API/Shared/Wallets.hs

```diff
@@ -1443,17 +1444,22 @@ spec = describe "SHARED_WALLETS" $ do
                 }],
                 "passphrase": #{fixturePassphrase}
             }|]
-        (_, ApiFee (Quantity _) (Quantity feeMax) _ _) <- unsafeRequest ctx
+        (_, ApiFee (Quantity feeMin) (Quantity feeMax) _ _) <- unsafeRequest ctx
             (Link.getTransactionFeeOld @'Shelley wShelley) payloadTx
         let ep = Link.createTransactionOld @'Shelley
         rTx <- request @(ApiTransaction n) ctx (ep wShelley) Default payloadTx
         expectResponseCode HTTP.status202 rTx
+        expectField
```

Comment:
I don't see how this expectation is related to the test (that shared wallet can discover its address). I suggest dropping the expectation.

Reply:
We agreed to keep existing estimated-vs-real-fee checks until we've completed the transition to balanceTx and come up with adequate replacement for them (likely more unit-level goldens).

When I changed the balance check below to be based on `fee` rather than `feeMax` I _essentially_ removed a comparison between postTransaction and postTransactionFee. That's why I'm re-adding it.

I'd prefer to error on the side of having too many cheeks rather than too few.

Perhaps you'd be happier with links to a JIRA task to remove these checks? 🤔 

https://github.com/input-output-hk/cardano-wallet/pull/3827#discussion_r1160059375

Reply:
https://github.com/input-output-hk/cardano-wallet/pull/3839/commits/c582f24da0c93a689cee4c09343fe56f79ccd14c